mithril-aggregator/src/database/migration.rs

```diff
@@ -456,10 +470,14 @@ create table new_certificate (
     foreign key (parent_certificate_id) references certificate(certificate_id)
 );
 insert into new_certificate
+        (  certificate_id, parent_certificate_id, message, signature, aggregate_verification_key,
+        epoch, beacon, protocol_version, protocol_parameters, protocol_message, signers,
+        initiated_at,
+        sealed_at)
     select certificate_id, parent_certificate_id, message, signature, aggregate_verification_key,
         epoch, beacon, protocol_version, protocol_parameters, protocol_message, signers,
-        strftime('%Y-%m-%dT%H:%M:%fZ', initiated_at) as initiated_at,
-        strftime('%Y-%m-%dT%H:%M:%fZ', sealed_at) as sealed_at
+        strftime('%Y-%m-%dT%H:%M:%fZ', initiated_at),
```

Comment:
why remove the field aliases?

Reply:
They don't matter in these `insert into xxx select`, here the argument is purely positional, the name is ignored.